pkg/cke/main.go

```diff
@@ -19,12 +20,13 @@ import (
 )
 
 var (
-	flgHTTP            = pflag.String("http", "0.0.0.0:10180", "<Listen IP>:<Port number>")
-	flgConfigPath      = pflag.String("config", "/etc/cke/config.yml", "configuration file path")
-	flgInterval        = pflag.String("interval", "1m", "check interval")
-	flgCertsGCInterval = pflag.String("certs-gc-interval", "1h", "tidy interval for expired certificates")
-	flgSessionTTL      = pflag.String("session-ttl", "60s", "leader session's TTL")
-	flgDebugSabakan    = pflag.Bool("debug-sabakan", false, "debug sabakan integration")
+	flgHTTP                 = pflag.String("http", "0.0.0.0:10180", "<Listen IP>:<Port number>")
+	flgConfigPath           = pflag.String("config", "/etc/cke/config.yml", "configuration file path")
+	flgInterval             = pflag.String("interval", "1m", "check interval")
+	flgCertsGCInterval      = pflag.String("certs-gc-interval", "1h", "tidy interval for expired certificates")
+	flgSessionTTL           = pflag.String("session-ttl", "60s", "leader session's TTL")
+	flgDebugSabakan         = pflag.Bool("debug-sabakan", false, "debug sabakan integration")
+	flgMaxConcurrentUpdates = pflag.Int("max-concurrent-updates", 10, "the maximum number of components that can be updated simultaneously")
```

Comment:
```suggestion
	flgMaxConcurrentUpdates = pflag.Int("max-concurrent-updates", 10, "the maximum number of worker-node components that can be updated simultaneously")
```

The description above seems too simple.
It would be nice to add an explanation for how CKE uses this parameter.
How about updating the following section?

https://github.com/cybozu-go/cke/blob/v1.25.3/docs/cluster_overview.md#worker-nodes

CKE deploys each worker-node component concurrently on multiple nodes.
You can control the deployment concurrency by specifying the `max-concurrent-updates` command-line option.